code/data_structures/linked_list/linked_list/linked_list.rb

```diff
@@ -0,0 +1,111 @@
+# Part of Cosmos by OpenGenus Foundation
+class Node
+	attr_accessor :data
+	attr_accessor :next
+
+	def initialize(data)
+		@data = data
+		@next = nil
+	end
+end
+
+class LinkedList
+	attr_accessor :head
+	attr_accessor :size
```

Comment:
You should add a member function for the size. Don't want the client code modifying it, which may mess up results (unless that doesn't happen in Ruby, since I don't know the language very well.